libbroker/broker/internal/core_actor.cc

```diff
@@ -1137,8 +1124,7 @@ caf::result<caf::actor> core_actor_state::attach_master(const std::string& name,
   auto in = self
               ->make_observable() //
               .from_resource(con2)
-              .map([this](const command_message& msg) {
-                metrics_for(packed_message_type::command).buffered->Increment();
+              .map([this](const command_message& msg) { //
```

Comment:
Errant `//`, same below

Reply:
I think these are clang-format "overrides". If you remove them, the layout works out quiet differently. I'll steer clear for now, but thank you for flagging. :+1:

Reply:
Yep, those are to force `clang-format` to leave that line break alone. Otherwise, `clang-format` makes some questionable choices here that make the code harder to read IMHO. I'm sure there are ways to tweak the `clang-format` config to have it heave better in such cases but I didn't have the time yet to play around with it.